core/controllers/base_test.py

```diff
@@ -1661,17 +1661,18 @@ def test_handler_with_schemas_should_not_be_present_in_schema_requiring_handler_
         in payload validator only contains handler class names which require
```

Comment:
test_handlers_with_schemas_are_not_in_handler_schema_todo_list or similar should get you within the line limit, I think.

Reply:
Done